client/packages/common/src/ui/layout/tables/components/Cells/NumberInputCell/NumberCell.tsx

```diff
@@ -31,7 +31,7 @@ export const NumberCell = <T extends RecordWithId>({
         padding: '4px 8px',
       }}
     >
-      <Tooltip title={value.toString()}>
+      <Tooltip title={value?.toString()}>
```

Comment:
@mark-prins can you please change `as number` above to be `as number | undefined`

Reply:
Good point - have also corrected the other two usages of `value` based on the expected typing